ReactWindows/ReactNative.Shared/Bridge/IReactContext.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// Context wrapper for the React instance to manage lifecycle events.
+    /// </summary>
+    public interface IReactContext : IDisposable

```

Comment:
>IReactContext [](start = 21, length = 13)

Is this interface really necessary?  1) ReactContext has a few internal methods that are being used.  2) Interfaces are only useful so far as someone could actually be able to implement them.